crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -518,13 +545,13 @@ impl Entities {
 
 #[derive(Copy, Clone, Debug)]
 pub struct EntityMeta {
-    pub generation: u32,
+    pub generation: Option<NonZeroU32>,
```

Comment:
What does this being `None` mean? I think as discussed it's that the slot is 'dead', but please document that here